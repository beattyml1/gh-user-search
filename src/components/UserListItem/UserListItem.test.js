import * as React from 'react';
import ReactDOM from 'react-dom';
import {UserListItem} from './UserListItem';
import renderer from 'react-test-renderer';

it('creates expected mark up for 0 results', () => {
    const tree = renderer.create(
        <UserListItem login={"dev"} name={"Dev"}  location={"Pittsburgh"} avatarUrl={""}
                      followers={2} following={2} starred={3} publicRepositories={4}
                      repositoriesLink={""} followersLink={""} followingLink={""} starredLink={""}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot()
});