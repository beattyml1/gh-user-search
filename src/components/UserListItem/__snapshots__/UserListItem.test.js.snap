// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`creates expected mark up for 0 results 1`] = `
<div
  className="user"
>
  <a
    href={undefined}
  >
    <img
      alt="@dev Avatar"
      className="avatar"
      src=""
    />
  </a>
  <span
    className="user-info"
  >
    <h3>
      <a
        className="user-login"
        href={undefined}
      >
        dev
      </a>
      <span
        className="user-name"
      >
        Dev
      </span>
    </h3>
    <div
      className="location"
    >
      Pittsburgh
    </div>
    <div
      className="user-stats"
    >
      <a
        className="publicRepositories"
        href=""
      >
        4 public repositories
      </a>
      <a
        className="followers"
        href=""
      >
        2 followers
      </a>
      <a
        className="following"
        href=""
      >
        2 following
      </a>
      <a
        className="starred"
        href=""
      >
        3 starred
      </a>
    </div>
  </span>
</div>
`;
